docs/external-etcd.md

```diff
@@ -0,0 +1,77 @@
+# Support for external etcd
+
+Cluster API Bootstrap Provider Kubeadm  supports using an external etcd cluster for your workload Kubernetes clusters.
+
+### ⚠️ Warnings ⚠️
+
+Before getting started you should be aware of the expectations that come with using an external etcd cluster.
+
+* Cluster API is unable to manage any aspect of the external etcd cluster.
+* Depending on how you configure your etcd nodes you may incur additional cloud costs in data transfer.
+    * As an example, cross availability zone traffic can cost money on cloud providers. You don't have to deploy etcd
+    across availability zones, but if you do please be aware of the costs.
```

Comment:
It applies to more than cross-AZ traffic with CAPA, especially if using the default VPC config, but I'm probably getting a bit too deep in the weeds for this doc :)